Frontend/src/views/WalksDetail.vue

```diff
@@ -105,6 +124,9 @@ export default {
     }).then(response => {
       this.$store.commit(SET_PHOTOWALK,response.data);
       this.isLoading = false;
+      this.photowalkTitle = response.data.name;
+      this.photowalkRegion = response.data.region;
+      this.photowalkLength = response.data.length;
```

Comment:
Warum werden diese drei Datensätze extra abgespeichert? Die werden doch vorher ebenfalls in den Store commited und können daraus ausgelesen werden?